enhancements/console/pinned-resources.md

```diff
@@ -0,0 +1,179 @@
+---
+title: pinned-resources
+authors:
+  - "@debsmita1"
+reviewers:
+  - "@jerolimov"
+  - "@JoelSpeed"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2022-11-15
+last-updated: 2022-11-15
+tracking-link:
+  - https://issues.redhat.com/browse/ODC-5012
+status: implementable
+---
+
+# Define pre-pinned resources
+
+## Summary
+
+The Developer perspective on the OpenShift Console provides pinned resources on the navigation which enables the user to go to the resource’s list page with a single click.
+
+Some customers want to provide defaults for those pre-pinned resources for all users.
+
+On the Developer perspective navigation, users can see the default pinned resources if they have not customized these pinned resources. The customers have no way to set these default pinned resources for their users.
+
+This enhancement allows cluster admins to define pre-pinned resources for new users and users who have not customized their navigation items.
+
+![Default pinned resources](default-pinned-resources.png)
+
+## Motivation
+
+Some customers want to provide default pre-pinned resources for all users.
+
+### User Stories
+
+#### Story 1
+
+Customer wants to customize the developer perspective navigation by providing pre-pinned resources to users who do not have their navigation items customized.
+
+As a cluster admin, I want to provide default pre-pinned resources for all users.
+
+### Goals
+
+- Allow cluster admins to define default pre-pinned resources on the dev perspective navigation
+
+### Non-Goals
+
+N/A
+
+## Proposal
+
+We can extend the existing `operator.openshift.io/v1` / `Console CRD`.
+We can add `spec.customization.perspectives.pinnedResources` which lists default pre-pinned resources
+
+### Workflow Description
+
+To add default pre-pinned resources:
+
+```text
+├── spec
+│   ├── customization
+|       ├── perspectives
+|           ├── id
+|           ├── ...
+|           ├── pinnedResources
+|               └── group
+                └── version
+                └── kind
+└── ...
+```
+
+Each perspective conforms to the following schema :
+
+- `pinnedResources: configv1.GroupVersionKind[]` , defines a list of default pinned resources
+- `group: string` , defines the group of the resource type.
+- `version: string` , defines the version of the resource type.
+- `kind: string` , defines the kind of the resource type.
+
+If the `pinnedResources` list is empty, the default pinned resources will be shown.
+If the `pinnedResources` list is non-empty, only the listed items will be shown if the user hasn't already customized the navigation items.
+
```

Comment:
Can we add one more sentence here that the UI doesn't support pinned resources for the `"admin"` perspective, like:

```suggestion
Note: The Administrator perspective (id="admin") doesn't support pinned resources at the moment and can be excluded via CEP validations.
```